src/stylesheets/components/_footer.scss

```diff
@@ -196,13 +196,15 @@ footer .usa-link svg[class*="svg-inline"] {
       @include u-text("secondary-dark");
       @include u-text("bold");
       @include u-display("block");
+      @include u-margin-y(2);
     }
 
     .sds-feedback__input {
-      height: 150px;
-      width: 100%;
-      border: none;
+      @include u-height('card');
```

Comment:
Height can be set in `textarea` elements by setting the number of rows. This will keep the feedback's height independent of card size. For example 
```html
<textarea id="feedback" class="sds-feedback__input" rows="8"></textarea>

Reply:
Done.